tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -41,6 +41,25 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.ServiceVersion"/>
         public const string ServiceVersion = "DD_VERSION";
 
+        /// <summary>
+        /// Configuration key for the application's git repo URL. Sets the "_dd.git.repository_url" tag on every <see cref="Span"/>.
+        /// </summary>
+        /// <seealso cref="TracerSettings.ServiceVersion"/>
+        public const string GitRepositoryUrl = "DD_GIT_REPOSITORY_URL";
+
+        /// <summary>
+        /// Configuration key for the application's git commit hash. Sets the "_dd.git.commit.sha" tag on every <see cref="Span"/>.
+        /// </summary>
+        /// <seealso cref="TracerSettings.ServiceVersion"/>
+        public const string GitCommitSha = "DD_GIT_COMMIT_SHA";
+
+        /// <summary>
+        /// Configuration key for enabling the tagging of every telemetry event with git metadata.
+        /// Default is value is true (enabled).
+        /// </summary>
+        /// <seealso cref="TracerSettings.TraceEnabled"/>
```

Comment:
```suggestion
        /// <seealso cref="TracerSettings.GitMetadataEnabled"/>
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/commit/989c9c823ebb86da47f43eeaf48a2466a04e38af